crates/sys/src/lib.rs

```diff
@@ -20,6 +21,9 @@ mod inlines;
 #[allow(non_upper_case_globals)]
 #[allow(non_camel_case_types)]
 #[allow(non_snake_case)]
+#[allow(clippy::transmute_int_to_bool)]
+#[allow(clippy::useless_transmute)]
```

Comment:
It might make sense to `#[allow(clippy::all)]`, since it's unlikely there's much we can do about any warnings in bindgen code.

I think the equivalent for rustc lints is `#[allow(warnings, unused)]`.

Reply:
Eh, what we have here is sufficient for closing the current warnings, and if something new comes up it is probably worth us noticing so we can open an issue on `bindgen`.

Reply:
By the way, would not

```suggestion
#[allow(
    non_upper_case_globals,
    non_camel_case_types,
    non_snake_case,
    clippy::transmute_int_to_bool
)]
```

look more compact here?

Reply:
Indeed. Done.